next.config.js

```diff
@@ -1,8 +1,36 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  distDir: 'build',
+  distDir: "build",
+  experimental: {
+    appDir: true
+  },
+  //this is to support automatic camelCasing in module.scss files
+  webpack: config => {
```

Comment:
this webpack config allows us to reference css classes/ids in camelCase at the component level